src/components/internal/DatePicker.tsx

```diff
@@ -8,7 +8,7 @@ import "./DatePicker.css";
 export interface DatePickerProps {
   value?: Date;
   onSelect: (value: Date) => void;
-  disabledDays?: Modifier;
+  disabledDays?: Modifier[];
```

Comment:
What do you think about `Modifier | Modifier[]` to allow for either?

Reply:
I had that initially, but Brandon suggested updating to only accept an array.

Reply:
Haha, I had just asked to see if we can only do the array type. I'm not sold on one way or the other, so happy to move back to what you originally had @ksk5280.

Reply:
Ha I don't have a strong opinion - my original intention was to just forward this prop and not deviate from `react-day-picker`'s implementation so the docs would still make sense with our wrapper... so this was a miss from me that I made it more restrictive

Reply:
I think it seems good to keep it flexible, so I reverted back to `Modifier | Modifier[]`